e2e/volumegroupsnapshot_base.go

```diff
@@ -456,6 +456,22 @@ func (v *volumeGroupSnapshotterBase) testVolumeGroupSnapshot(vol VolumeGroupSnap
 		return fmt.Errorf("failed to create volume group snapshot: %w", err)
 	}
 
+	// Create and delete 5 additional group snapshots.
+	for i := range 5 {
```

Comment:
We should have this as a configurable value in the base structure and set it only for RBD so that we don't run the test when it is not required.